nfc/libs/NfcCoreLib/lib/Nci/phNciNfc_Init.c

```diff
@@ -848,136 +768,4 @@ phNciNfc_ResetNtfCb(void*     pContext,
     }
     PH_LOG_NCI_FUNC_EXIT();
     return wStatus;
-}
-
-static NFCSTATUS phNciNfc_RegAllNtfs(void*     pContext)
```

Comment:
Thanks for pointing out. This function got moved firstly in order to be sure we were not going to miss the first NCI_CORE_RESET_NTF after power on or HW reset. 

Moving the phNciNfc_RegAllNtfs call to phNciNfc_Initialize help to ensure that notification callback are registered before the first sequence execution.

If moving back phNciNfc_RegAllNtfs to phNciNfc_InitReset which is in gphNciNfc_Init sequence will cause phNciNfc_Reinitialize to re-execute phNciNfc_RegAllNtfs and trigger the need back for bSkipRegisterAllNtfs.

Also i believe, phNciNfc_Init.c contains NCI sequence for the NFCC initialization.